### PR TITLE
Respect primitve mapping on type and format attributs of 1.2 spec

### PR DESCRIPTION
**Pull request by frodrigo**
Respect primitve mapping on type and format attributs of 1.2 swagger spec: https://github.com/swagger-api/swagger-spec/blob/master/versions/1.2.md#431-primitives

### CODE REVIEW

File: lib/grape-swagger/doc_methods.rb [hunk 1]
```diff
@@ -1,5 +1,15 @@
 module GrapeSwagger
   module DocMethods
+    @@primitive_mapping = {
```
Comment (dblock) [inline lib/grape-swagger/doc_methods.rb]: This can be a CONSTANT.

**Comment by dblock** [comment]
Thanks. This needs tests and a passing build, please.

**Comment by frodrigo** [comment]
All known problem fixed.

File: lib/grape-swagger/doc_methods.rb [hunk 2]
```diff
@@ -1,5 +1,15 @@
 module GrapeSwagger
   module DocMethods
+    PRIMITIVE_MAPPING = {
```
Comment (dblock) [inline lib/grape-swagger/doc_methods.rb]: Maybe PRIMITIVE_MAPPINGS (plural)?

**Comment by dblock** [comment]
It would be nice to extend tests to all primitive mappings, but I would merge without. Can you squash the commits, please?

**Comment by frodrigo** [comment]
Squash and plural done.

**Comment by dblock** [comment]
Merged, thanks.

**Pull request merged**